agent/api/task/task.go

```diff
@@ -94,6 +94,8 @@ type TaskOverrides struct{}
 type Task struct {
 	// Arn is the unique identifier for the task
 	Arn string
+	//AvailabilityZone is the zone for task locates in
+	AvailabilityZone string
```

Comment:
why is this needed? as I understand, we use agent's availabilityZone field for v2/v3 responses, right?